### DISCUSSION TITLE
Add "NetBox Deployment To AWS in One Click" to the wiki

### DISCUSSION THREAD

**Discussion by igor-raits** [netbox-community/netbox#14308] [Ideas]
Hello folks,

I've been experimenting with NetBox deployment on AWS with using cloud services and not caring about EC2 maintenances, etc. Definitely in some simple and reliable way.

So I've written https://medium.com/gooddata-developers/netbox-deployment-to-aws-in-one-click-ea3704f13f29 and would appreciate if you could add it onto https://github.com/netbox-community/netbox/wiki/Community-Contributions page (I guess somewhere under "Alternative Installations" section).

Thank you!

**Comment by igor-raits** [comment]
@jeremystretch any chance to make this happen? Thanks.

**Comment by sierrasoleil** [comment]
Have you tried deploying this to EKS instead of ECS? I've been trying to use the community Helm chart, but I can't get the RDS auth to work, even with SSL disabled.
